test/src/module/config/parseTest.c

```diff
@@ -602,6 +626,8 @@ testRun(void)
             sizeof(optionResolveOrder) / sizeof(ConfigOption), CFG_OPTION_TOTAL,
             "check that the option resolve list contains an entry for every option");
 
```

Comment:
Per discussion with David, I will take a look at this to resolve.

Reply:
The problem is the test, on it's own, does not set the command to backup so line 629 https://github.com/pgbackrest/pgbackrest/pull/1221/files#diff-7fddaa2d1c17b169e0ffb5074d9882fc3b0177119b8710c792b1b7edf124d649R629 - should be removed since it is redundant with a test at line 1195 https://github.com/pgbackrest/pgbackrest/pull/1221/files#diff-7fddaa2d1c17b169e0ffb5074d9882fc3b0177119b8710c792b1b7edf124d649R1195 further in the same run.  I will commit the fix if you agree.

Reply:
Looks good to me.

Reply:
Fixed in d63233ed62f.